source/reference/command/setIndexCommitQuorum.txt

```diff
@@ -78,7 +78,7 @@ The command takes the following fields:
        marks the ``indexes`` as ready.
        
        Starting in MongoDB v5.0, it's possible to resume some
-       :ref:`interupted index builds<index-operations-build-failure>`
+       :ref:`interrupted index builds<index-operations-build-failure>`
```

Comment:
Oh, this isn't directly related to your fix, but our common practice is to include a space between the `ref` text and link:

```suggestion
       :ref:`interrupted index builds <index-operations-build-failure>`
```

This won't affect the rendered view, and just makes things slightly easier to read when editing.

Reply:
Got it, thanks!